packages/ingest/test/queue.js

```diff
@@ -66,7 +66,8 @@ test('the queue receives a correctly formatted workflow message', async(t) => {
         },
         payload: { granules: [granule] }
       };
-
+      t.not(actualMessage.cumulus_meta.execution_name);
```

Comment:
Also test that the arn is returned from line 48 and that with a pdr the expected message is correct